.gitignore

```diff
@@ -22,3 +22,6 @@ src/workflow_app/django_nscd_workflow.egg-info/
 # Sphinx Doc related
 _build/
 *.log
+
+# Environment related
+.envrc
```

Comment:
This should be in your [global gitignore](https://github.com/peterfpeterson/dotfiles/blob/master/gitexcludes) and [configure it](https://github.com/peterfpeterson/dotfiles/blob/master/gitconfig#L19-L21).

Reply:
actually my intent was to put it here because other people also use `.envrc` and I wanted to save them the trouble of having to exclude it themselves. Do you still think it should be up to each person?